moz_ingest/io_modules/decoders/moz_ingest/common.lua

```diff
@@ -21,6 +21,11 @@ decoders_moz_ingest_common = {
 
     -- String used to specify GeoIP city database location on disk.
     city_db_file = "/usr/share/geoip/GeoIP2-City.mmdb", -- optional, if not specified no city/country geoip lookup is performed
+    city_report_geoname_id = true, -- optional, if not specified then geoname_id isn't reported
```

Comment:
> it should become more configuration driven
like this?

Reply:
I was thinking more like the logging inputs. https://mozilla-services.github.io/lua_sandbox_extensions/maxminddb/io_modules/maxminddb/heka.html  but lets just file a different issue for that.

Reply:
okay. do i remove this option or leave it in?

Reply:
Created #287.  We can remove the above unless we don't want the subdivision info in the generic ingestion data in the interim @mreid-moz?